### ISSUE TITLE
ArrayIndexOutOfBoundsException for ALIA formula

### ISSUE THREAD

**Issue by rainoftime** [ultimate-pa/smtinterpol#73]
Hi, for the following formula,
[arridx.txt](https://github.com/ultimate-pa/smtinterpol/files/4488411/arridx.txt)

smtinterpol commit 0610d35 throws an exception 
~~~~
WARN - Quant: Clause contains literal that is not almost uninterpreted: (= .q13.7 .q14.8)
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
Unexpected Exception: java.lang.ArrayIndexOutOfBoundsException: 1
(error "java.lang.ArrayIndexOutOfBoundsException: 1")
~~~~

**Comment by rainoftime** [comment]
Here is another test case
[array2.txt](https://github.com/ultimate-pa/smtinterpol/files/4492616/array2.txt)

**Comment by tanjaschindler** [comment]
This should be fixed by commit 47ca378
